content/sensu-go/5.15/reference/checks.md

```diff
@@ -189,7 +196,7 @@ metadata:
   namespace: default
 spec:
   command: check-cpu.sh -w 75 -c 90
-  cron: '* * * * *'
+  cron: 0 * * * *
```

Comment:
this is correct for yaml 👍

Reply:
Got it. I changed the 0 back to * so it would be every minute on the minute as we describe.